app/mailers/scheduled_publish_mailer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class ScheduledPublishMailer < ApplicationMailer
+  self.delivery_job = EmailDeliveryJob
+
+  add_template_helper(EditionUrlHelper)
+
+  def publish_email(edition, user)
```

Comment:
Might want to call this `success_email` since we're likely to have a `failure_email` in this class?

Reply:
Yep, good shout!